main.js

```diff
@@ -0,0 +1,379 @@
+    [null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null]];
+var columnNumber = null;
+var playerModeToggle = 1;
+
+function togglePlayerMode() {
+    $('.title').text("Third Player Summoned. Player One: Select Your Deity.")
+    $('.togglePlayerNumber').text("TWO PLAYER MODE");
+    playerModeToggle = 1 - playerModeToggle; //if playerModeToggle === 0; three player mode activates; else two player mode;
+    if (playerModeToggle === 1) {
+        $('.title').text("Third Player Sacrificed. Player One: Select Your Deity.");
+        $('.togglePlayerNumber').text("THREE PLAYER MODE");
+    }
+    toggleAICount = 0;
+}
+
+var toggleAISwitch = 1; //toggle that switches between player and AI
+var toggleAICount = 0;
+function toggleAI() { //toggles whether AI should be on/off
+    toggleAICount = 1 - toggleAICount;
+    $('.title').text("Choose your color and play with the Gods.");
+    if (toggleAICount === 0 ) {
+        $('.title').text("Player One: Choose Your Diety");
+    }
+    playerModeToggle = 1;
+    $('.togglePlayerNumber').text("THREE PLAYER MODE");
+};
+
+//Function to assign image class to players
+function playerColor() {
+    if (playerSwitch === 2) {
+        if (playerOneColor === null) {
+            playerOneColor = $(this).css("background-image");
+        }
+    } else if (playerSwitch === 1) {
+        if (playerTwoColor === null) {
+            playerTwoColor = $(this).css("background-image");
+        }
+    } else if (playerSwitch === 0 && playerModeToggle === 0) {
+        if (playerThreeColor === null) {
+            playerThreeColor = $(this).css("background-image");
+        }
+    }
+    $(this).addClass("gray");
+    $(this).off("click");
+    $(".toggleAI, .togglePlayerNumber").hide();
+    playerSwitch--;
+    if (playerSwitch === 1 && toggleAICount === 0) {
+        $('.title').text("Player Two: Choose Your Diety");
+    } else if (playerSwitch === 0 && toggleAICount === 0) {
+        $('.title').text("Player Three: Choose Your Diety");
+    }
+    if (toggleAICount === 1 && victoryTrigger === 0) { //if AI mode is on, runs function that allows AI to randomly choose color
+        aiSelectColor();
+    };
+    if (playerOneColor != null && playerTwoColor != null && playerThreeColor != null && toggleAICount === 0 || playerOneColor != null && playerTwoColor != null && toggleAICount === 1 || playerOneColor != null && playerTwoColor != null && playerModeToggle === 1) {
+        $('.choose-color-page').hide();
+        $('.game_area').show();
+        $('.gameTitle').text("Player One's Turn");
+        playerSwitch = 2;
+    }
+}
+
+function aiSelectColor() { // Allows AI to pick random color after player one chooses color
+    var colorArray = [$(".red").css("background-image"), $(".blue").css("background-image"), $(".green").css("background-image"), $(".green").css("background-image")];
```

Comment:
This includes all options, including what the player chose. This leads to the unfortunate possibility that both the user and the AI are using the same piece, which leads to a very confusing game, trust me 😂. Make sure you restrict this array to only include the options the player did not choose.